proposals/stage-0/TRAVEL_FUND/README.md

```diff
@@ -0,0 +1,57 @@
+# OpenJS Travel Fund
+>  Stage 0
+
+Tracked by https://github.com/openjs-foundation/cross-project-council/issues/172
+
+## Champion
+
+Jonah Stiennon @jonahss
+
+## Description
+
+The Node.js organizations that existed prior to the formation of the OpenJS foundation oversaw a monetary fund for the purpose of member and affiliate travel to, and lodging at, events related to the foundation's mission. This proposal proposes moving the management responsibilities of this fund to the Cross Project Council (CPC), and making the funds available to all members of this broader organization. Since all projects have voting representation on the CPC, the beneficiaries of the fund will jointly manage it, and the previous owners of the fund will still vote on its use.
+
+We shall review the process in this proposal, but the intent is to leave the mechanics of the fund unchanged, simply moving from the purview of the Node.js project into the domain of the OpenJS Foundation, overseen by the CPC. This operation can be viewed as a refactor which moves the travel fund up one level in the organizational tree.
+
+The current fund is used mostly for attending collaborator summits, and we do not expect this to change.
+
+## Difference from the current process
+
+The existing process requires approval from two members of the [Node.js TSC](https://github.com/nodejs/TSC) and two members of the [Node.js Community Committee](https://github.com/nodejs/community-committee).
+The new process will require approval from four members of the CPC.
```

Comment:
What if the approvals all come from members who do not work on the project which the requester is affiliated with?

Will there be a limit for each project? If there isn't, is it possible that one project may end up using all of the funds, leaving very little for other projects? Or if there is, is it possible that one project may have their limit reached without having their members looking into their own allocations?

Reply:
Great questions. Here are my thoughts so far, but others should chime in:

> What if the approvals all come from members who do not work on the project which the requester is affiliated with?

That should be fine. All requesters will be somehow affiliated with one of the projects, and the approvals come from members who collectively represent all the projects. Since the CPC is already balanced to not have too much representation from any one project, we shouldn't have to worry about one project having an unbalanced amount of control.
Also, the current wording for the fund outlines the potential for vetoes as well as approvals.

>Will there be a limit for each project?

When it comes to allocations per project, I'd like to avoid such a system until it looks like we're going to need one. As of now, the travel fund is predominantly used for Node.js Interactive and Collaborator Summits. The [considerations for request approval](https://github.com/nodejs/admin/blob/master/MEMBER_TRAVEL_FUND.md#considerations-for-approval-of-a-request) will not change, so preference will still be given to official OpenJS events such as the collaborator summit.  With the fund shared by more people, I can see the budget may need to be raised to pay for more people to attend those events. 

>is it possible that one project may end up using all of the funds, leaving very little for other projects?

It looks like, among the impact project, only Appium and jQuery have independent events, while the rest of the projects typically host events at large javascript conferences. It is interesting to note that, had a member of Node.js asked for travel funds to attend these events, they would have qualified. If you think it necessary, we could add a clause to the [Equity](https://github.com/nodejs/admin/blob/master/MEMBER_TRAVEL_FUND.md#equity) section of the considerations for approval, which could dictate preference given to projects which have not already had members who received funding recently.

Approvals to fund travel to events other than collaborator summits should not be allowed to drain the budget so that at the end of the year, it cannot fund travel to said summits. How is this currently prevented in the current organization of the fund?

Reply:
>Approvals to fund travel to events other than collaborator summits should not be allowed to drain the budget so that at the end of the year, it cannot fund travel to said summits. How is this currently prevented in the current organization of the fund?

This is currently up to the members of the TSC/CommComm to manage.  So far the requests have been limited in number so it has not been hard for the TSC/CommComm to manage that.  For a fund serving more projects it might be more complicated...

Reply:
It sounds like performing a quick count of the number of members being added here would help in planning for the addition. Would you like me to do that and get some round numbers to help us prepare in an informed way?

Reply:
The word "preference" is used in this thread and in the current policy. How is that word defined in this context?

Reply:
Additionally, you reference a file in the nodejs/admin repo. Is there a reason that's not also included in this PR to this repo, since it is being up-leveled? 🤔

Reply:
> How is this currently prevented in the current organization of the fund?

I don't think we had turn down most requests for events other than the collaboration summit. Some non-summit requests that I could remember:

https://github.com/nodejs/admin/pull/195
https://github.com/nodejs/admin/pull/162

Reply:
@joyeecheung Makes a lot of sense. I assumed we could keep the same process, but maybe we should formalize it more, since the pool of people may be larger, and the original system may just have been lucky to not have an issue so far.

I don't want to over-engineer, but how about I take a census of the current pool of people who can draw on the fund, and an estimate of the new size of the population. I can also build a spreadsheet which estimates the amount of money to set aside for the summits, in order to guide the decision makers?

Reply:
@Jonahss That sounds great! I think we just need to at least have some numbers to keep the budget under control - at least as someone who used to approve these requests from time to time, it's really difficult to tell whether the request that I was reading was going to affect the whole budget or not, unless it was sent after the Interactive near the end of the year. We did a funding survey for the Interactive last year which helped us having an idea about the expected balance, it would be very useful if a survey like that can be done before allocating the budget from foundation.

Reply:
On a side note, it should be quite helpful if the survey includes questions about whether the requester are up for room-sharing (and any preference with their roommates), as that should cut down the lodging cost significantly compared to people booking the rooms separately. (it would also be nice if they could stay in the same or close hotels as it's safer and gives you more time to socialize with other attendees). But yeah this is just some idea from past experience, not a blocker for the proposal :)

Reply:
That's great context, thanks!

Yes I think for the next stage, I'll go out and survey all the projects for their anticipated use of travel funds.

Reply:
@joyeecheung good ideas.  I think this is a good time to discuss new ideas/ways of doing things. Not that it should block this landing as stage-0, but good to have those ideas discussed as it gets broader discussion and moves to later stages.

Reply:
In the next PR, I'll include the documents in question, and then we can review and edit those in the review :)